tests/sentry/api/endpoints/relocations/test_index.py

```diff
@@ -69,16 +69,65 @@ def test_success_simple(self, uploading_complete_mock):
                                 ).getvalue(),
                                 content_type="application/tar",
                             ),
-                            "orgs": ["testing", "foo"],
+                            "orgs": "testing, foo",
                         },
                         format="multipart",
                     )
 
         assert response.status_code == 201
         assert Relocation.objects.count() == relocation_count + 1
         assert RelocationFile.objects.count() == relocation_file_count + 1
+        assert Relocation.objects.get(owner_id=self.owner.id).want_org_slugs == ["testing", "foo"]
         assert uploading_complete_mock.call_count == 1
 
+    # pytest parametrize does not work in TestCase subclasses, so hack around this
+    for org_slugs, expected in [
+        ("testing,foo,", ["testing", "foo"]),
+        ("testing,,foo", ["testing", "", "foo"]),
```

Comment:
`""` isn't a valid org name. We should check org names against `src/sentry/api/fields/sentry_slug.py::ORG_SLUG_PATTERN`.

Reply:
done

Reply:
Yeah, but we should fail as quickly as possible - waiting until the file is uploaded to GCP, then re-read and scanned for org matching, is both poor UX and wasteful of ingress/egress resources. If we get an invalid org slug, we already know that the relocation will fail, so we should just tell the user to submit the form again right away.